lib/bolt/shell/powershell/snippets.rb

```diff
@@ -55,14 +55,19 @@ def run_script(arguments, script_path)
             }
             #{build_arg_list}
 
+            if([string]::IsNullOrEmpty($invokeArgs.ScriptBlock)){
+              $Host.UI.WriteErrorLine("Error: Failed to obtain scriptblock from '#{script_path}'. Running scripts may be disabled on this system. For more information, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170");
+              exit 1;
+            }
```

Comment:
@puppetlabs/bolt I need help with the error message here.

This checks if `$invokeArgs.ScriptBlock` is null or empty and tells the user the likely cause is execution policy. However there could be other reasons, but I won't be able to tell why exactly. Is there a better way to phrase this?

Reply:
You could link to a page in our docs with a more in depth explanation. These are always hard problems.
would "The system may have an execution policy that prevents running unsigned scripts"  be slightly more informative?

Reply:
I think as long as you include `may be` (like you do here) this is fine. We do this a few other places: lib/bolt/shell/bash.rb#L448, lib/bolt/transport/ssh/connection.rb#L233.

Reply:
I think @kreeuwijk addition addresses @adreyer  suggestion nicely, and @lucywyman suggestion goes in line with the edit @beechtom provided